Install/default-layouts/layout0.json

```diff
@@ -4,35 +4,35 @@
     "Visible": true,
     "Position": {
       "X": 0.0,
-      "Y": 0.017642908
+      "Y": 0.017361112
     },
     "Size": {
       "X": 0.553125,
-      "Y": 0.98447424
+      "Y": 0.9840278
     }
   },
   {
     "Title": "Settings",
     "Visible": false,
     "Position": {
```

Comment:
Please avoid committing files that are not relevant for actual change.
This will make reviewing easier and avoid side effects and regressions.

Reply:
Ah. i think these changes were caused by git branching interference. I can clean this up locally.